vscode/src/completions/inline-completion-item-provider.ts

```diff
@@ -385,6 +386,12 @@ export class InlineCompletionItemProvider
                     isDotComUser: this.config.isDotComUser,
                 })
 
+                // Do not increment the `preFinalCancellationCheck` counter if the result is empty.
+                // We don't have an opportunity to show a completion if it's null.
```

Comment:
You use two different phrases, `empty` and `null`, to refer to the same state. It appears that you're making a distinction, which is confusing. Don't use different words unless you want to make a distinction.
```suggestion
                // Do not increment the `preFinalCancellationCheck` counter if the result is empty.
                // We don't have an opportunity to show a completion if it's empty.
```